swagger-spec/files/versions_list.yaml

```diff
@@ -1 +1,60 @@
 # /files/{file_id}/versions/
+get:
+  summary: Retrieve file versions
```

Comment:
"List all file versions"

I've been inconsistent about this (sorry), but I think I've fixed it everywhere now. List views should use "List..." and detail views should use "Retrieve..."